UIKitDZ/UIKitDZ/ViewControllers/PayViewController.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  PayViewController.swift
+//  UIKitDZ
+//
+//  Created by coder on 29.09.2022.
+//
+
+import UIKit
+/// Экран с оплатой
+final class PayViewController: UIViewController {
+    // MARK: Properties
```

Comment:
```suggestion
    // MARK: Properties
```
точно они все public должны быть?